src/test/resources/com/arpnetworking/logback/StenoEncoderTest.testEncodeArrayWithException.json

```diff
@@ -1 +1 @@
-{"time":"1970-01-01T00:00:00.000Z","name":"logEvent","level":"info","data":{},"exception":{"type":"java.lang.NullPointerException","message":"npe!","backtrace":[],"data":{}},"context":{"host":"<HOST>","processId":"<PROCESS_ID>","threadId":"<THREAD_ID>"},"id":"<ID>","version":"0"}
+{"time":"1970-01-01T00:00:00.000Z","name":"logEvent","level":"info","data":{},"exception":{"type":"java.lang.NullPointerException","message":"npe!","backtrace":[],"data":{"_id":"<ID>","_class":"java.lang.NullPointerException"}},"context":{"host":"<HOST>","processId":"<PROCESS_ID>","threadId":"<THREAD_ID>"},"id":"<ID>","version":"0"}
```

Comment:
I'm not sure I follow. The Steno format -- not defined by us -- had an exception block, but some exception data overflows into the catch-all data block. The class field in the data block is there due to our object serialization standards. Removing it only applies to exceptions serialized as Steno exceptions. 

Exceptions serialized in the top level data block, or as values of members in the data block of Steno exceptions _should_ retain the class field (because they lack the Steno exception type field). Representing this context of when to expect type and when to expect class seems complicated -- especially to users trying to deserialize such a json blob.

I can give you a call and we can discuss further; perhaps I'm not seeing what you're saying.